spec/author_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../authors/author'
+
+describe Author do
+  before :each do
+    @author = Author.new('Christian', 'Raga')
+  end
+
```

Comment:
- [ ] As required for the project, kindly make sure all the implanted methods in each class(all public methods in all classes) have unit tests. e.g.: The author class: add_item method should be tested, The Book class: can_be_archived method should be tested,...
Make sure you apply in all classes. 
```ruby
# example of testing one the methods in a class
describe Author do
  before(:all) do
    @author = Author.new('John', 'Doe')
  end

  context '#initialize' do
    it 'should initialize a new author' do
      expect(@author.first_name).to eq('John')
      expect(@author.last_name).to eq('Doe')
      expect(@author.items).to eq([])
    end
  end

  context '#add_item' do
    it 'should add the item to author' do
      item = Item.new(Date.today, false)
      @author.add_item(item)
      expect(@author.items).to eq([item])
    end
  end
en
```

Reply:
Tests added